lib/dialects/mssql/query-generator.js

```diff
@@ -339,37 +340,38 @@ class MSSQLQueryGenerator extends AbstractQueryGenerator {
 
     if (allAttributes.length > 0) {
       attrValueHashes.forEach(attrValueHash => {
-        tuples.push(`(${
-          allAttributes.map(key =>
-            this.escape(attrValueHash[key])).join(',')
-        })`);
+        const values = allAttributes.map(key => {
+          if (attrValueHash[key] instanceof Utils.SequelizeMethod) {
+            return this.handleSequelizeMethod(attrValueHash[key]);
+          }
+
+          if (attrValueHash[key] === null || attrValueHash[key] === undefined) return 'NULL';
```

Comment:
@sushantdhiman do you have any preference on single line returns, imo forcing them to always be blocks is easier to read.

Reply:
Before v5 is released let's setup `prettier` so everything looks good